scripts/generate/main.ts

```diff
@@ -42,14 +47,22 @@ const maintainersCodeOwners = () => {
       return `/styles/${slug} ${codeOwners}`;
     })
     .join("\n");
-};
-const userstylesStaffCodeOwners = () => {
+}
+async function userstylesStaffCodeOwners() {
   const paths = ["/.github/", "/scripts/", "/template/"];
-  return paths.map((path) => `${path} @catppuccin/userstyles-staff`).join(
+
+  const octokit = getAuthenticatedOctokit();
+  const staffMembers = await getUserstylesTeamMembers(
+    octokit,
+    "userstyles-staff",
+  );
+  return paths.map((path) =>
+    `${path} ${staffMembers.map((member) => "@" + member).join(" ")}`
```

Comment:
This should have the codeowners file directly tag each of us staff individually instead of as a team, as mentioned a while back.

![CleanShot 2024-12-29 at 10 54 09](https://github.com/user-attachments/assets/32060a40-963c-4d12-8f09-9a65294db547)